### ISSUE TITLE
Mismatch of links and missing documentation for enable/disable

### ISSUE THREAD

**Issue by w0rmr1d3r** [pi-hole/docs#1019]
### Versions
Issue is in docs: https://docs.pi-hole.net/core/pihole-command/
Currently at 5.18.2

### Platform
PiHole official docs

### Expected behavior

_A clear and concise description of what you expected to happen._

Link for `Enable Disable` in table works in table:
https://docs.pi-hole.net/core/pihole-command/

Documentation for `Enable` is missing in the docs.

### Actual behavior / bug

_A clear and concise description of what the bug is._

The link in the initial table for the pihole command for Enable/Disable doesn't work. The link:
![Screen Shot 2024-04-18 at 17 01 37 PM](https://github.com/pi-hole/docs/assets/20722014/851a4633-eb39-45bd-b199-eb2871a26a2e)
Goes to: https://docs.pi-hole.net/core/pihole-command/#enable-&-disable

But the section, from the menu on the right:
![Screen Shot 2024-04-18 at 17 05 58 PM](https://github.com/pi-hole/docs/assets/20722014/46032477-dafb-4b5a-a4c8-0c035a9c5651)
Is: https://docs.pi-hole.net/core/pihole-command/#enable-disable

Also, the instructions only appear for `Disable` while the section is for both `Enable` and `Disable`

### Steps to reproduce

1. Go to https://docs.pi-hole.net/core/pihole-command/
2. On the main table, the link for `Enable & Disable`, if clicked doesn't work
3. Click on the link for `Enable & Disable` on the menu on the right goes to the section
4. Now on the section, the description is only for the `disable` not for the `enable` command.

## Debug Token

Doesn't apply.

### Screenshots

_If applicable, add screenshots to help explain your problem._

Besides the one above, adding the screenshot for the section on Enable Disable, only displaying info on Disable:
![Screen Shot 2024-04-18 at 17 01 46 PM](https://github.com/pi-hole/docs/assets/20722014/7d5e9739-9a95-4b50-810f-2aac91937c08)

### Additional context

_Add any other context about the problem here._

**Comment by XhmikosR** [comment]
This seems to be fixed in the latest docs @DL6ER (I guess an update to mkdocs which properly strips `&` from anchor links)

**Comment by yubiuser** [comment]
The links works now.

But OP is still right there is no example of `pihole enable`

**Comment by yubiuser** [comment]
> Now on the section, the description is only for the disable not for the enable command.

Yes, it's a trivial one line addition and then I think the issue can be cloased
